newt/builder/paths.go

```diff
@@ -31,6 +31,10 @@ import (
 const BUILD_NAME_APP = "app"
 const BUILD_NAME_LOADER = "loader"
 
+func ProjectRoot() string {
+	return project.GetProject().Path() + "/"
```

Comment:
Just a nitpick, but I prefer when path strings do not include a trailing slash.  Inevitably, people construct a bigger path by appending "/rest/of/path", and the result is a double slash in the middle of the string.  Also, the other paths in `paths.go` do not include a trailing slash.

Reply:
removed trailing '/' in the latest PS